test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperDropDownListExtensionsTest.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TestCommon;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Core
+{
+    public class HtmlHelperDropDownListExtensionsTest
+    {
+        private static readonly List<SelectListItem> BasicSelectList = new List<SelectListItem>
+        {
+            new SelectListItem { Text = "Zero",  Value = "0"},
+            new SelectListItem { Text = "One",   Value = "1"},
+            new SelectListItem { Text = "Two",   Value = "2"},
+            new SelectListItem { Text = "Three", Value = "3"},
+        };
+
+        [Fact]
+        public void DropDownList_UsesSpecifiedExpression()
```

Comment:
`DropDownList_FindsSelectList()`. This test isn't using the specified expression. It finds the select list at the named location in VDD and that's it. Actually a case we have a bug about -- somewhere in the backlog -- because users can't put both the expression and the select list in VDD.